src/chain/config.ts

```diff
@@ -1,22 +1,41 @@
+import * as admin from "firebase-admin";
+import { Event } from "@polkadot/types/interfaces";
+
+export type InterestedEvent = { pattern: string, getPushData: (event: Event) => admin.messaging.TopicMessage };
+
 const kusamaNetwork = {
-  ws: 'wss://kusama.api.onfinality.io/public-ws',
-  events: [],
+  ws: "wss://kusama.api.onfinality.io/public-ws",
+  events: [] as InterestedEvent []
 } as const;
 
 const polkadotNetwork = {
-  ws: 'wss://rpc.polkadot.io',
-  events: [],
+  ws: "wss://rpc.polkadot.io",
+  events: [] as InterestedEvent[]
 } as const;
 
 const litentryNetwork = {
-  ws: 'wss://3.0.201.137',
-  events: [],
+  // ws: 'wss://3.0.201.137',
+  ws: "wss://staging.registrar.litentry.io",
+  events: [{
+    pattern: "treasury.Proposed",
+    getPushData() {
+      return {
+        topic: "treasury.Proposed",
+        data: { deeplink: "litentry://treasury" },
+        notification: {
+          title: `New Treasury Proposal`,
+          body: "A new treasury proposal has been submitted, check it out!"
+        }
+      };
+    }
+  }] as InterestedEvent[]
```

Comment:
Since we are using `as const` typescript will infer the correct type, so we don't need to do as InterestedEvent.. but probably would be better if we have a type like:

```
type Network = {
  ws: string;
  events: Array<InterestedEvent>
}
```
And then do: 
```
const litentryNetwork: Network = {..} 
const polkadotNetwork: Network = {...}
...
```